Wrapping/Generators/Python/itk/__init__.py.in

```diff
@@ -25,6 +25,7 @@
 import os
 import sys
 
+__version__ = "@ITK_VERSION_MAJOR@.@ITK_VERSION_MINOR@.@ITK_VERSION_PATCH@"
```

Comment:
Is `PythonSuffix` available here? Should it go into this version string?

Reply:
Unfortunately, the suffix is not available here.